capframex@1.7.4: download via aria2 failed

```
> scoop update capframex
capframex: 1.7.2 -> 1.7.4
Updating one outdated app:
Updating 'capframex' (1.7.2 -> 1.7.4)
Downloading new version
Starting download with aria2 ...
Download: 12/31 16:01:26 [ERROR] CUID#8 - Download aborted. URI=https://github.com/CXWorld/CapFrameX/releases/download/vDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/CXWorld/CapFrameX/releases/download/v1.Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: ee2183|ERR |       0B/s|C:/Users/test/scoop/cache/capframex#1.7.4#86ca54c.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/CXWorld/CapFrameX/releases/download/v1.7.4/release_1.7.4_portable.zip
    referer=https://github.com/CXWorld/CapFrameX/releases/download/v1.7.4/
    dir=C:\Users\test\scoop\cache
    out=capframex#1.7.4#86ca54c.zip

ERROR & 'C:\Users\test\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\test\scoop\cache\capframex.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=25892 --continue --summary-interval=0 --auto-save-interval=1 --all-proxy='localhost:2080'

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Extras/issues/new?title=capframex%401.7.4%3a+download+via+aria2+failed
```

--- a/bucket/capframex.json
+++ b/bucket/capframex.json
@@ -8,8 +8,8 @@
     },
     "architecture": {
         "64bit": {
-            "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.7.4/release_1.7.4_portable.zip",
-            "hash": "e0580075628a5c39f71e4b3c57523819b70f2796586de35f00342b5d622e786c"
+            "url": "https://github.com/CXWorld/CapFrameX/releases/download/v1.7.4_release/release_1.7.4_portable.zip",
+            "hash": "bcda7067baf42272803a1bf3d5c4c54b869dbc5d9e9f5a49afa4f15c19e62ede"
         }
     },
     "bin": "CapFrameX.exe",
@@ -23,7 +23,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/CXWorld/CapFrameX/releases/download/v$version/release_$version_portable.zip"
+                "url": "https://github.com/CXWorld/CapFrameX/releases/download/v$version_release/release_$version_portable.zip"
             }
         }
     }

